spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  before(:each) do
+   @or_dmv = DmvDataService.new.or_dmv_office_locations
+   @f_factory = FacilityFactory.new
+  end
+
+  it "exists" do
+    expect(@f_factory).to be_an_instance_of(FacilityFactory)
+  end
+
+  it "can create facilities from OR database" do
```

Comment:
This test should be passing the `@or_dmv` data to your `@f_factory`. 
So you have this in your `before(:each)`
```ruby
@f_facotry = FacilityFactory.new
@org_dmv = DmvDataService.new.or_dmv_office_locations
```
Then in your test you should have
```ruby
expect(@f_factory.or_facility_creator(@or_dmv)).to be_an(Array)
expect(@f_factory.or_facility_creator(@or_dmv).first.name).to be_a(String)
...
```
That is how you are getting the `name`, `address`, `phone`.